Python/Pomodoro_Timer/README.md

```diff
@@ -0,0 +1,26 @@
+# Pomodoro Timer
+
+- The python script implements a productivity Pomodoro Timer which involves 25 minutes of high focus intervals broken by 5 minutes break between each one of them.
+- The script also allows a website blocking functionality for Windows OS users to increase focus, which get automatically unblocked during the break time.
+
+## Requirements:
+
+- Inorder to enable the website blocking functionality, you have to run this script as an Administrator in the command prompt.
+
+## Running the script:
+
```

Comment:
Can you also mention here, that users can change the websites to be blocked as well, in case you use a file to read the websites (read other review comments), mention that here in the steps as well.